docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,119 @@
+- From the wrong account, go to **[Receipts](https://expo.dev/accounts/[account]/settings/receipts)** and initiate a [request for a refund](/billing/invoices-and-receipts/#request-a-refund).
+
+### I am on a Free plan and only need a few extra builds or updates
+
+If you are on a Free plan and have completed your monthly quota of free builds and updates, upgrade to the [On-demand plan](/billing/plans/). It offers a pay-as-you-go option for EAS Build and the same limits as the Free plan for EAS Update. Once your requirements are fulfilled, you can always downgrade to the Free from the On-demand plan.
+
+To upgrade from the Free plan to the On-demand plan, see [Upgrade to a new plan](/billing/manage/#upgrade-to-a-new-plan).
+
+To downgrade from an On-demand to a Free plan, see [Cancel a plan](/billing/manage/#cancel-a-plan).
+
+## Billing
+
+### When does a billing cycle start for a plan?
+
+For the Free plan, the billing cycle starts on the first day of the calendar month.
+
+For all paid plans, the billing cycle starts from the subscription date of that plan.
+
+### How do I update my billing information or add a tax ID?
+
+To update your Organization account's billing information or add a tax ID, make sure that you have either an [**Owner** or **Admin** role](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role. After confirming your role:
+
+- Go to your [**Account or Organization settings** > **Billing**](https://expo.dev/settings/billing), and click on **Update billing information**. This will take you to Stripe's portal.
+- On Stripe's portal, under **Billing information**, click on **Update information** to update billing-related information such as billing name, email, address, and tax ID.
+
+See [Manage billing information](/billing/manage/#manage-billing-information) for more details.
+
+### Can I update the billing info on my last invoice?
+
+No. Updating billing information will only be reflected on the next invoice.
+
+### Can you email me the receipts?
+
+No. Your account's Owner or Admin can download them. See [Download a receipt](/billing/invoices-and-receipts/#download-and-view-a-receipt) for more information.
+
+### How can I reduce the amount of EAS Build usage by using EAS Update?
+
+Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. For native code changes, you can also automate the build process using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime).
```

Comment:
```suggestion
Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team.
```
Thinking we strike this for now and look at updating it once we have more specific advice and workflows for using Fingerprint to switch between builds and updates. Or, we could reframe this as a "coming soon: use fingerprint to selectively build or update depending on whether you already have a compatible native runtime", particularly in reference to the last few sentences of the blog.

Reply:
I've updated this with the context you provide.

Can also create a follow up task for adding fingerprint context.